lib/transformers/transformers.js

```diff
@@ -48,7 +51,44 @@ const mapInReq = ({ ctx, health, bodyKeys, bodyKeysRegex, bodyKeysCallback, head
   });
 };
 
-const mapInRes = (res, req, startedAt, reqId) => {
+const mapOutRes = ({ ctx, health, bodyKeys, bodyKeysRegex, bodyKeysCallback, headersRegex, maxBodyValueChars }) => {
```

Comment:
This is mainly a reordering so that server logs are together and request logs are together